lib/src/main/kotlin/com/swmansion/starknet/data/types/Contracts.kt

```diff
@@ -2,78 +2,8 @@ package com.swmansion.starknet.data.types
 
 import com.swmansion.starknet.extensions.base64Gzipped
 import kotlinx.serialization.*
-import kotlinx.serialization.descriptors.SerialDescriptor
-import kotlinx.serialization.encoding.Decoder
-import kotlinx.serialization.encoding.Encoder
 import kotlinx.serialization.json.*
 
-enum class AbiEntryType {
-    FELT,
-    FELT_ARRAY,
-    STRING
-}
-
-@Serializable
-data class AbiEntry(
-    val name: String,
-    val type: AbiEntryType,
-)
-
-@Serializable
-sealed class AbiElement
-
```

Comment:
all that is no longer used due to deploy remove, right?

Reply:
It wasn't used by any part of library even before I removed it. I think these got declared a while ago and finally were never used in any module

Reply:
This will actually come in handy in rpc 2.1, so let's leave it. I will remove the unnecessary parts (if there will be some)